src/twisted/python/bytes.py

```diff
@@ -0,0 +1,38 @@
+# -*- test-case-name: twisted.python.test.test_bytes -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Utility for converting between L{unicode} and L{bytes}
+"""
+
+from __future__ import division, absolute_import, print_function
+
+from twisted.python.compat import unicode
+
+
+def ensureBytes(s, encoding="ascii", errors="strict"):
+    """
+    Convert L{unicode} string to L{bytes}.
```

Comment:
maybe the docstring should add a warning that this should not be used for py2/py3 porting.